lib/config.js

```diff
@@ -22,6 +22,12 @@ var autoconfig = function(overrides) {
       "sync_frequency" : SYNC_FREQUENCY,
       "storage_strategy": "dom",
       "do_console_log": false
+    },
+    syncFields: {
+      result: ['id', 'stepResults'],
```

Comment:
IMHO I think that placement of this is little bit unfortunate and I think that the best will be to place this information directly into mongoose module as this problem is mongoose specific and having this code in demo app breaking typical code abstraction layers. 

Each schema can have set of fields and his custom hashing function for sync.